src/Network/Http/Response.php

```diff
@@ -388,7 +388,7 @@ protected function _getJson()
             return $this->_json;
         }
         $data = json_decode($this->_body, true);
-        if ($data) {
+        if (is_array($data)) {
```

Comment:
I don't think there's a need to check return value type at all since return type of `json_decode` is mixed.